yew/src/callback.rs

```diff
@@ -64,10 +106,10 @@ impl<IN> Callback<IN> {
     /// This method calls the callback's function.
     pub fn emit(&self, value: IN) {
         match self {
-            Callback::Callback(cb) => cb(value),
-            Callback::CallbackOnce(rc) => {
+            Callback::Callback { cb, .. } => cb(value),
+            Callback::Once(rc) => {
                 let cb = rc.replace(None);
-                let f = cb.expect("callback in CallbackOnce has already been used");
+                let f = cb.expect("callback in Once has already been used");
```

Comment:
This message is slightly confusing, how about:
```suggestion
                let f = cb.expect("callback contains `FnOnce` which has already been used");
```